python/matlab_converter.py

```diff
@@ -0,0 +1,23 @@
+from analysis import *
+from utils import *
+import matplotlib.pyplot as plt 
+import numpy as np 
+import matplotlib.animation as animation
+import os
+import pandas as pd
+import scipy.io
+
+os.chdir("./")
+
+filename = './result/Simulation Results/'
```

Comment:
Have you used [argparse](https://docs.python.org/3/library/argparse.html) before? Maybe it's better to create a command-line argument for this instead of hardcoding a path in the file.